### PR TITLE
Add a per document dust limit

### PR DESCRIPTION
**Pull request by dzen**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -50,6 +51,7 @@ Here it is in context:
     .. reviewer-meta::
         :written-on: 1974-05-19
         :proofread-on: 1974-06-20
+        :dust_days_limit: 25
```
Comment (lburg) [inline README.rst]: Prefixing with `dust_` makes sense as a global setting, we can drop it here though. I'd rename the parameter `proofread-frequency`, which seems clearer in this context

**Review by lburg — COMMENTED** [review]
Great idea! I'd rather name the parameter another way, otherwise it seems good

File: README.rst (hunk 1 above)
Comment (dzen) [reply README.rst]: We'll, we've got multiple names for this same mecanism so as the configuration or project name.
I would prefer to keep one naming for this.

**Comment by lburg** [comment]
I corrected a few renaming oversights and merged :)

**Pull request closed without merging**